analysis/topEFT/datacard_maker.py

```diff
@@ -50,14 +50,36 @@ def read(self):
         if len(self.coeffs)==0: self.coeffs = self.hists['njets']._wcnames
 
         # Get list of channels
-        self.ch2lss = list({k[1]:0 for k in self.hists['ptbl'].values().keys() if '2lss' in k[1]})
-        self.ch2lss += list({k[1]:0 for k in self.hists['njets'].values().keys() if '2lss' in k[1]})
-        self.ch2lss_p = list({k[1]:0 for k in self.hists['ptbl'].values().keys() if '2lss_p' in k[1]})
-        self.ch2lss_p += list({k[1]:0 for k in self.hists['njets'].values().keys() if '2lss_p' in k[1]})
-        self.ch2lss_m = list({k[1]:0 for k in self.hists['ptbl'].values().keys() if '2lss_m' in k[1]})
-        self.ch2lss_m += list({k[1]:0 for k in self.hists['njets'].values().keys() if '2lss_m' in k[1]})
+        self.ch2lss = list({k[1]:0 for k in self.hists['ptbl'].values().keys() if '2lss' in k[1] and not '4t' in
+                                                                                                            k[1]})
+        self.ch2lss += list({k[1]:0 for k in self.hists['njets'].values().keys() if '2lss' in k[1] and not '4t' in
+                                                                                                            k[1]})
+        self.ch2lss_p = list({k[1]:0 for k in self.hists['ptbl'].values().keys() if '2lss_p' in k[1] and not '4t' in
+                                                                                                            k[1]})
+        self.ch2lss_p += list({k[1]:0 for k in self.hists['njets'].values().keys() if '2lss_p' in k[1] and not '4t' in
+                                                                                                            k[1]})
+        self.ch2lss_m = list({k[1]:0 for k in self.hists['ptbl'].values().keys() if '2lss_m' in k[1] and not '4t' in
+                                                                                                            k[1]})
+        self.ch2lss_m += list({k[1]:0 for k in self.hists['njets'].values().keys() if '2lss_m' in k[1] and not '4t' in
+                                                                                                            k[1]})
```

Comment:
This is a fairly minor comment, but just to keep the formatting consistent with the current formatting of this file, I'm wondering we might want to keep these statements all in one line (instead of splitting them up across two lines). In general I think splitting things up across multiple lines can sometimes improve readability, but since the preexisting code does not do this, I think I'd probably lean in the direction of trying to be consistent with the current formatting.

Reply:
Agreed, I thought my editor was only visually introducing linebreaks and not physically as well.